core/src/main/java/com/novoda/accessibility/ActionsAlertDialogCreator.java

```diff
@@ -8,28 +8,25 @@
 public class ActionsAlertDialogCreator {
 
     private static final int NO_TITLE = 0;
-    
+
     private final Context context;
 
     @StringRes
     private final int title;
 
-    private final Actions actions;
-
-    public ActionsAlertDialogCreator(Context context, Actions actions) {
-        this(context, NO_TITLE, actions);
+    public ActionsAlertDialogCreator(Context context) {
+        this(context, NO_TITLE);
     }
 
-    public ActionsAlertDialogCreator(Context context, @StringRes int title, Actions actions) {
+    public ActionsAlertDialogCreator(Context context, @StringRes int title) {
         this.context = context;
         this.title = title;
-        this.actions = actions;
     }
 
-    public AlertDialog create() {
+    public AlertDialog create(final Actions actions) {
         AlertDialog.Builder builder = new AlertDialog.Builder(context);
```

Comment:
have you considered passing this in or... *drumroll* that and a static factory method?